AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/ItineraryController.cs

```diff
@@ -128,7 +127,7 @@ public ItineraryController(IMediator mediator, IItineraryEditModelValidator itin
         [Route("Admin/Itinerary/{id}/[Action]")]
         public async Task<IActionResult> SelectRequests(int id)
         {
-            var orgId = await _mediator.SendAsync(new OrganizationIdQuery { ItineraryId = id });
+            var orgId = await GetOrganizationIdBy(id);
```

Comment:
Noticed that getting organization ID and verifying that it's a valid one or that the user can administer the given organization appears multiple times in several actions. Question: would it justify to convert that code into custom action filter?